sonic_platform_base/sfp_base.py

```diff
@@ -52,24 +52,21 @@ def get_transceiver_bulk_status(self):
         ========================================================================
         keys                       |Value Format   |Information	
         ---------------------------|---------------|----------------------------
-        RX LOS                     |BOOLEAN        |RX lost-of-signal status,
-                                   |               |True if has RX los, False if not.
-        TX FAULT                   |BOOLEAN        |TX fault status,
-                                   |               |True if has TX fault, False if not.
-        Reset status               |BOOLEAN        |reset status,
-                                   |               |True if SFP in reset, False if not.
-        LP mode                    |BOOLEAN        |low power mode status,
-                                   |               |True in lp mode, False if not.
-        TX disable                 |BOOLEAN        |TX disable status,
-                                   |               |True TX disabled, False if not.
-        TX disabled channel        |HEX            |disabled TX channles in hex,
-                                   |               |bits 0 to 3 represent channel 0
+        rx_los                     |BOOLEAN        |RX lost-of-signal status, True if has RX los, False if not.
```

Comment:
typo: "lost-of-signal" should be "loss-of-signal". This existed before your changes, but please fix.

Reply:
fixed.